packages/web-app-draw-io/src/App.vue

```diff
@@ -64,7 +64,7 @@ export default defineComponent({
   watch: {
     currentFileContext: {
       handler: function () {
-        this.load()
+        this.checkPermissions()
```

Comment:
No need to call `load()` here, it will be called later anyways. `checkPermissions()` is sufficient.